### ISSUE TITLE
problem with the methods in .jsh file

### ISSUE THREAD

**Issue by sophia820** [nus-cs2030/2223-s1#25]
# Summary

I imported import java.util.ArrayList; to use the .get(int index)
but, when I compile, the error occurs. How should I solve this problem?

## Description:

Clarifications or details with respect to the issue y
<img width="446" alt="Screenshot 2022-09-03 at 12 13 08 PM" src="https://user-images.githubusercontent.com/110775510/188255426-e7cec032-a1e6-4138-8e49-037962f2241b.png">
ou are facing

## Code Snippets

Attach code snippets (if any)

## Screenshots

Screenshots (if any)

**Comment by wilsonwid** [comment]
I don't think you are allowed to use the Java `ArrayList` class (CodeCrunch will complain), and aside from that, I feel like your error is caused by your implementation of `Point` as an array instead of a `List`.

**Comment by danqiye1** [comment]
This issue is stale because it has been open for 30 days with no activity.
